### PR TITLE
MTM-58684: databroker multi-structure-change fix RN 2024

### PR DESCRIPTION
**Pull request by gapa-c8y**
Prod core code merged when its dev version is 10.18.540.244, which should be the lowest possible release version.

Issue: https://cumulocity.atlassian.net/browse/MTM-58684 
Core PR: https://github.com/Cumulocity-IoT/cumulocity-core/pull/3119

### CODE REVIEW

**Review by rohitjoshi-c8y — APPROVED** [review]

File: content/change-logs/platform-services/cumulocity-10.18.540.244-databroker-multiple-structure-change-fowarding-fix.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+---
+date: 2024-10-17
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10.18.540.244-databroker-multiple-structure-change-fowarding-fix.md]:
```suggestion
date: 
```

File: content/change-logs/platform-services/cumulocity-10.18.540.244-databroker-multiple-structure-change-fowarding-fix.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  component:
+- value: component-V6J_FcOT2
+  label: Data broker
+  build_artifact:
+- value: tc-QHwMfWtBk7
+  label: cumulocity
+  ticket: MTM-58684
+  version: 10.18.540.244
+---
+When a data broker connection synchronizes a structural change that adds multiple child assets or devices to a group, all assets or devices are now sent immediately. Previously, only one asset or device would sent immediately and others would not be synchronized until the periodic consistency check next ran. This behaviour is most commonly seen when assigning multiple assets or devices in a Device management groups page, when the group is within the scope of a data broker connection filter.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10.18.540.244-databroker-multiple-structure-change-fowarding-fix.md]:
```suggestion
When a data broker connection synchronizes a structural change that adds multiple child assets or devices to a group, all assets or devices are now sent immediately. Previously, only one asset or device was sent immediately and others were not synchronized until the periodic consistency check next ran. This behaviour is most commonly seen when assigning multiple assets or devices in a Device Management groups page, when the group is within the scope of a data broker connection filter.
```

File: content/change-logs/platform-services/cumulocity-10.18.540.244-databroker-multiple-structure-change-fowarding-fix.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+---
+date: 
+title: "All children now synchronized in immediate data broker structural change forwarding"
+product_area: Platform services
+change_type:
+- value: change-VSkj2iV9m
+  label: Fix
+  component:
```
Comment (gapa-c8y) [inline content/change-logs/platform-services/cumulocity-10.18.540.244-databroker-multiple-structure-change-fowarding-fix.md]:
```suggestion
```

Comment (gapa-c8y) [reply content/change-logs/platform-services/cumulocity-10.18.540.244-databroker-multiple-structure-change-fowarding-fix.md]: This seems to be breaking the Hugo build - the few others I checked don't have it

File: content/change-logs/platform-services/cumulocity-10.18.540.244-databroker-multiple-structure-change-fowarding-fix.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10.18.540.244-databroker-multiple-structure-change-fowarding-fix.md]: The front matter section is still not correct. Please compare to other files, must be:

image

**Review by pink-mongoose — COMMENTED** [review]
@gapa-c8y the explanation is brilliant for a techie but for a standard user they will not understand what change in behaviour they will see.  Can we re-word it to the effect that users will no longer see a delay when adding multiple child assets ....

File: content/change-logs/platform-services/cumulocity-10-18-540-244-databroker-multiple-structure-change-fowarding-fix.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+---
```
Comment (gapa-c8y) [inline content/change-logs/platform-services/cumulocity-10-18-540-244-databroker-multiple-structure-change-fowarding-fix.md:1]:
```suggestion
```

**Review by BeateRixen — APPROVED** [review]

**Review by schmiel-cumulocity — APPROVED** [review]

**Review by JanePorter — APPROVED** [review]

**Comment by BeateRixen** [comment]
@gapa-c8y Can this PR be merged? Otherwise it will not get a date automatically when the cumulocity version is 10.18.540.244

**Pull request merged**